### DISCUSSION TITLE
BTCPay 2.0 General feedback

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#6294] [General]
Hey everyone,

Woah, 2.0 took a lot of time on our end. Congratulations to the team and contributors for pushing it through. We're dedicating entire Q4 to listening to your feedback about 2.0 and BTCPay in general. 

Let us know what you love about it, what you hate, what frustrates you or which feature would make you extremely happy.

We'll carefully review all suggestions and try to roadmap the most upvoted ones. Please try not to duplicate suggestions if you can and feel free to upvote/downvote requests and help us get a better sense of the direction for 2025! 🎉

**Comment by Fenix2013** [comment]
Hi, I am currently using BTCPay Server version 1.13.5 and yesterday I wanted to update it to the new version 2.0.0, but the system does not allow me to do so.I use the BTCPay Server by connecting through LunaNode , so I contacted them and asked them for help me with the update (it happened to me in the past that I couldn't update to the latest version and they helped me update).However, today they replied that they cannot help me yet, here is their exact answer: 

"The update hasn't been merged into master at https://github.com/btcpayserver/btcpayserver-docker yet so you'll just need to wait for them, you can ask about the progress at https://chat.btcpayserver.org if you want."

Yesterday I joined your official channel at https://chat.btcpayserver.org and I was looking for advice.Ty you answered me twice.You answered me twice. First to ssh in my server and run the following commands:

git fetch
git  checkout 2,0
btcpay-update.sh

Then I explained to you that I contacted Lunanode Support for help and they told me that it was necessary to wait if it will be merged at master branch at https://github.com/btcpayserver/btcpayserver-docker.When I asked you how long it may take, you answered me a second time: "You Can Update Regardless, in 1 Month." (look at atached photos).

![photo_2024-11-01_14-27-50](https://github.com/user-attachments/assets/7c6fa1b6-5434-4ef0-b12a-275d9bc4f31e)
![photo_2024-11-01_14-27-18](https://github.com/user-attachments/assets/b08dfc7c-c4d8-461b-8137-ad3aabc3214c)



I was very expected to have this update 2.0.0 (some of the plugins I use cannot be updated unless I use version 2.0.0-Dependencies not met >= 2.0.0),that's why I decided to  SSH in my server (I‘m using BTCPay Server via LunaNode in Windows 11 therefore I run SSh via Putty) and run the following commands:

         git fetch
         git checkout 2.0
         [btcpay-update.sh](http://btcpay-update.sh/)

but it was not possible (after entering the command I was notified that it is not possible to find  a git repository or any of the parent directories-look at atached photo).

![Snímka obrazovky 2024-11-01 130305](https://github.com/user-attachments/assets/61088350-5eb5-48fd-bf85-f9addf677bd6)


I think that unless it is not merged at master branch at https://github.com/btcpayserver/btcpayserver-docker,then will not be possible to make this update.What do you think about it?

**Comment by Fenix2013** [comment]
I resolved it,now I have the newest BTCPay Server version 2.0.0. This is the solution:

$ sudo su
$ cd/root/btcpayserver-docker
$ git fetch -a
$ git checkout 2.0
$ ./btcpay-update.sh

Thank you for your assistance and help!

![Snímka obrazovky 2024-11-01 175829](https://github.com/user-attachments/assets/57cd0850-175f-4296-b385-f631695dbdcb)

Kindest regards

**Reply by dennisreimann** [reply]
Thanks, I have [updated this in the blog post](https://blog.btcpayserver.org/btcpay-server-2-0/#-updating-to-20).

**Comment by iBobik** [comment]
Hi, I wanted to migrate to the v2, but I need to first migrate my LN wallets to the Nostr accounts and then upgrade BtcPay.

But how to enable Nostr plugin on the v1? Plugins page refuses to install it because current version requires BtcPay v2.
